submissions/Lu-sure/dom-api/index.js

```diff
@@ -0,0 +1,119 @@
+        'fondOf': 'science fiction, comic books, fan costumes, video games, his dog Cinnamon.',
+        'features': 'an Indian English accent, selective mutism social anxiety disorder, metrosexuality.'
+    },
+    {
+        'sideName': 'The astronaut here',
+        'name': 'Howard Wolowitz',
+        'img': 'img/howard.jpg',
+        'known': 'Howie, Froot Loops, Rocket Man, Wolowizard.',
+        'appearance': 'a bowl haircut, loud, vintage, 1960-era outfit, a shirt over a turtleneck or dickey on the neck, Vans or Converse sneakers, and skinny pants with massive belt buckle.', 
+        'quotes': '"I\'m a horny engineer... I never joke about math or sex."',
+        'fondOf': 'science fiction, comic books, fan costumes, video games, magic and card tricks.',
+        'features': 'multilingualism, not having a doctoral degree'
+    }
+];
+
+const body = document.querySelector('body');
+const side = document.getElementById('side');
+const main = document.getElementById('main');
+const sideButton = document.getElementById('side-button');
+
+const renderSide = function () {
+    const ul = document.createElement('ul');
+    for (let i=0; i<contentArray.length; i++) {
+        const sideHero = document.createElement('li');
+        sideHero.innerHTML = contentArray[i].sideName;
+        sideHero.classList.add('side-hero');
+        sideHero.setAttribute("data-idx", i);
+        ul.appendChild(sideHero);
+    };
+    ul.addEventListener('click', function (evt) {
+        changeContent(evt.target.getAttribute("data-idx"));
+        side.classList.remove('open');
+    });
+    side.appendChild(ul);
+};
+
+const render = function (index) {
+    const contentWrapper = document.createElement('div');
+    const content = contentArray[index];
+    if (!content) return;
+
+    const name = document.createElement('h2');
+    name.innerHTML = content.name;
+    name.classList.add('name');
+
+    const img = document.createElement('img');
+    img.setAttribute('src', content.img);
+    img.classList.add('image');
+
+    const known = document.createElement('h3');
+    known.innerHTML = `Also known as: ${content.known}`;
+    known.classList.add('known-for');
+
+    const appearance = document.createElement('p');
+    appearance.innerHTML = `Appearance: ${content.appearance}`;
+    appearance.classList.add('about-hero');
+
+    const quotes = document.createElement('p');
+    quotes.innerHTML = `Most famous quotes: ${content.quotes}`;
+    quotes.classList.add('about-hero');
+
+    const fondOf = document.createElement('p');
+    fondOf.innerHTML = `He is fond of: ${content.fondOf}`;
+    fondOf.classList.add('about-hero');
+
+    const features = document.createElement('p');
+    features.innerHTML = `Special features: ${content.features}`;
+    features.classList.add('about-hero');
+
+    contentWrapper.append(name, img, known, appearance, quotes, fondOf, features);
+    main.appendChild(contentWrapper);
+    contentWrapper.classList.add('hero');
```

Comment:
It's because there is another element in main block - the side button, which opens side-menu. So content-wrapper holds all hero info. Maybe it makes sense to move  out that button to outer tag?

Reply:
It is up to you. Seems like a good plan.

Reply:
Did it, LGTM.